modules/visual_script/doc_classes/VisualScriptBuiltinFunc.xml

```diff
@@ -190,36 +190,38 @@
 		<constant name="TEXT_PRINTRAW" value="58" enum="BuiltinFunc">
 			Print the given string to the standard output, without adding a newline.
 		</constant>
-		<constant name="VAR_TO_STR" value="59" enum="BuiltinFunc">
+		<constant name="TEXT_PRINT_VERBOSE" value="59" enum="BuiltinFunc">
+		</constant>
```

Comment:
Doesn't adding a new enum value here break compatibility for `.vs` scripts? I think it needs to be added to the end.